addons/messaging-executor/impl/src/main/java/org/jboss/windup/web/messaging/executor/MessagingProgressMonitor.java

```diff
@@ -129,6 +131,7 @@ public void subTask(String subTask)
         sendUpdate(execution);
 
         String message = String.format("[%d/%d] %s", execution.getWorkCompleted(), execution.getTotalWork(), subTask);
+        System.out.println(message);
```

Comment:
Why these duplicated additions of System.out.println?

Reply:
This was because of a misconfiguration of the logging and I needed a workaround for a demo. :)

I will remove them.